pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -300,3 +307,17 @@ func deploymentCopy(rc *kapi.ReplicationController) (*kapi.ReplicationController
 	}
 	return copied, nil
 }
+
+func (c *DeploymentConfigController) handleErr(err error, key interface{}) {
+	if _, isFatal := err.(fatalError); isFatal {
```

Comment:
Fatal errors are "forgotten" before the next resync.  This probably mirrored previous code, but I don't remember seeing a status update to the DC indicating that it fatally errored.  Do we do that?